### ISSUE TITLE
unrecognized  ingredient - lead

### ISSUE THREAD

**Issue by cyberbofh** [msarilar/EDEngineer#311]
found unrecognized  ingredient - lead

using EDEngineer v1.0.9.1

**Comment by msarilar** [comment]
thanks for reporting !
do you have more details about the ingredient? e.g. it's rarity and category?
Here's an example of the information that I need for each ingredient (this is Polonium):

```
  {
    "Name": "Polonium",
    "Rarity": "VeryRare",
    "FormattedName": "polonium",
    "Kind": "Material",
    "Subkind": "Raw",
    "OriginDetails": [ "Surface prospecting", "Mission reward" ],
    "Group": "Category5"
  }
```

**Comment by Hatsushira** [comment]
Raw materials grouping has changed.

`

Group | Tier 1 | Tier 2 | Tier 3 | Tier 4
-- | -- | -- | -- | --
1 | Carbon | Vanadium | Niobium | Yttrium
2 | Phosphorus | Chromium | Molybdenum | Technetium
3 | Sulphur | Manganese | Cadmium | Ruthenium
4 | Iron | Zinc | Tin | Selenium
5 | Nickel | Germanium | Tungsten | Tellurium
6 | Rhenium | Arsenic | Mercury | Polonium
7 | Lead | Zirconium | Boron | Antimony

`

**Comment by JohnXaverius** [comment]
{
    "Name": "Lead",
    "Rarity": "VeryCommon",
    "FormattedName": "lead",
    "Kind": "Material",
    "Subkind": "Raw",
    "OriginDetails": [ "Minning" ],
    "Group": "Category7"
  }
![born lead rhenium](https://user-images.githubusercontent.com/29731838/36945365-15c4dfbc-1fad-11e8-837f-45daf2d36038.jpg)

**Comment by msarilar** [comment]
fixed in 8845f903bb8f2a9ac42d14ab74ab4cf6f855581d (will be in next update)
